docs/DeveloperGuide.md

```diff
@@ -2,31 +2,76 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
-{:toc}
-
```

Comment:
I guess it was originally auto-generated. Have you checked if we can still rely on the auto-generated one?

Reply:
reverted back. AB3 uses a plugin of some sort called "markdown kramdown" for its toc function.